recipes/bempp-cl/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "bempp-cl" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bempp/bempp-cl/archive/v{{ version }}.tar.gz
+  sha256: 591817a2f724bfdda0e51073b99301db478090e778ab7b4d55f462e74b373c1b
+build:
+  number: 0
+  skip: True  # [py!=37]
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
Please remove the whole `build` section - when no compilers are used you don't need `build` in general.

Reply:
Done. Thanks for the comment.